packages/asset/contracts/AssetReveal.sol

```diff
@@ -65,19 +67,7 @@ contract AssetReveal is
     /// @param tokenId the tokenId of id idasset to reveal
     /// @param amount the amount of tokens to reveal
     function revealBurn(uint256 tokenId, uint256 amount) public {
```

Comment:
Should this function just be named burn now ?

Reply:
I agree with Woj

Reply:
Sounds good